### ISSUE TITLE
"State" thumbnail only showing when you're in the same folder as the file to print

### ISSUE THREAD

**Issue by JMVS** [jneilliii/OctoPrint-UltimakerFormatPackage#58]
Hello.

It seems I've found a problem when switching folder and having a file selected for print in which the thumnail disappears when you're not in the same folder as the file you've selected. I've tested this with two instances of OctoPrint & Cura Thumbnails and the results are the same.

OctoPrint: 1.7.3
Cura Thumnails: 1.0.2

**When you're in the same folder as the file selected for printing:**
![imagen](https://user-images.githubusercontent.com/7751148/166168557-1e3a05d4-001c-455a-b6e3-dce0b6b764a4.png)
![imagen](https://user-images.githubusercontent.com/7751148/166168583-419bc581-819b-48aa-a9e5-f2d84408f17b.png)

**When you have the same file selected but move to a different folder:**
![imagen](https://user-images.githubusercontent.com/7751148/166168573-980952ba-5d98-4de1-90b1-e839072db2c0.png)
![imagen](https://user-images.githubusercontent.com/7751148/166168589-09fb7536-6de8-4811-9033-8e17ad94a67a.png)

_Cura Thumbnails Settings:_
![imagen](https://user-images.githubusercontent.com/7751148/166168623-72094d60-ff8a-4721-8992-b05a20ead374.png)
![imagen](https://user-images.githubusercontent.com/7751148/166168629-d30b03d0-9fb2-4934-9fdf-b936b423563e.png)

Let me know if you want me to run any other tests.

**Comment by jneilliii** [comment]
Huh, I thought I had fixed this issue, but it may have been in my Slicer Thumbnails plugin instead. You could potentially try that one, as I've been considering retiring this one since that one supports most major slicers.

**Comment by fieldOfView** [comment]
But that does not integrate as automatically with the OctoPrint Connection plugin as this plugin does.

**Comment by jneilliii** [comment]
That's true, you have to enable the post-processing script manually, but everything after that works with the OctoPrint Connection plugin without issue.

**Comment by fieldOfView** [comment]
I'll see if I can directly support the other plugin as well (ie: include the thumbnail with the gcode if that plugin is detected)
